lib/lambda_ethereum_consensus/execution/execution_chain.ex

```diff
@@ -266,4 +269,15 @@ defmodule LambdaEthereumConsensus.Execution.ExecutionChain do
 
   defp slots_per_eth1_voting_period(),
     do: ChainSpec.get("EPOCHS_PER_ETH1_VOTING_PERIOD") * ChainSpec.get("SLOTS_PER_EPOCH")
+
+  defp persist_execution_state(state) do
+    put("", state)
+  end
```

Comment:
nit, maybe a one-liner here like the get?

Reply:
Done!